packages/calcite-components/src/utils/sortableComponent.ts

```diff
@@ -59,14 +57,14 @@ export interface SortableComponent {
   onDragSort: (event: Sortable.SortableEvent) => void;
 
   /**
-   * Element dragging started.
+   * Called when any sortable component drag starts.
    */
-  onDragStart?: (event: Sortable.SortableEvent) => void;
+  onDragStart: () => void;
 
   /**
-   * Element dragging ended.
+   * Called when any sortable component drag ends.
    */
-  onDragEnd?: (event: Sortable.SortableEvent) => void;
+  onDragEnd: () => void;
```

Comment:
Should we rename these to something like `enableListeners`? Since that is what they are basically for.